mmrazor/models/architectures/dynamic_ops/mixins/dynamic_mixins.py

```diff
@@ -270,8 +270,12 @@ def to_static_op(self: _BatchNorm) -> nn.Module:
 
         if running_mean is not None:
             static_bn.running_mean.copy_(running_mean)
+            static_bn.running_mean = static_bn.running_mean.to(
```

Comment:
?

Reply:
Without this line, the device of `running_mean` is cpu.
```
import torch.nn as nn
bn = nn.BatchNorm2d(3)
bn.cuda()
new_bn = nn.BatchNorm2d(3)
new_bn.running_mean.copy_(bn.running_mean)
print(new_bn.running_mean.device)
# cpu
```

Reply:
so, what will happen if the module generated by `to_static` is in "cpu" mode?

Reply:
All the other parameters are in "gpu" mode. So this will lead to a device error.

Reply:
Is this caused by the parameters in the loaded checkpoint?

Reply:
In [_dynamic_to_static](https://github.com/open-mmlab/mmrazor/blob/dev-1.x/mmrazor/structures/subnet/fix_subnet.py#L21), all dynamic modules are converted to static ones by `mutable.to_static_op()`. `running_mean, running_var, weight, bias` are all in GPU mode in `DynamicBatchNormMixin.to_static_op` during training.